akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -322,6 +329,11 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
   val -> = FSM.->
 
   /**
+   * Unicode alias for ->, matches a (S, S) pair
+   */
+  val → = FSM.->
```

Comment:
we can define new trait contains this method, and then let ``FSM`` inherit from it